lib/mapbox/geocode.py

```diff
@@ -171,6 +171,20 @@ def get_latlng(address, min_relevance_threshold=1):
     return latlng
 
 
+def extract_address_info(response):
```

Comment:
Does it make sense to get rid of information to go from more to less info?

Can we also include a link to the API docs in the docstring?

Reply:
Also, naming inconsistency.

Function is called with `results` but parameter name is `response`.